opentreemap/treemap/templates/treemap/partials/photo_social_media_sharing.html

```diff
@@ -0,0 +1,30 @@
+<div id="social-media-sharing-photo-upload-modal" class="modal fade">
+  <div class="modal-dialog">
+    <div class="modal-content">
+      <div class="modal-header">
+        <button type="button" class="close" data-dismiss="modal" aria-hidden="true">&times;</button>
+        <h2>Photo Added!</h2>
+      </div>
+      <div class="modal-body" data-class="share-links">
+        <p>Would you like to share this photo?</p>
+        <img id="social-media-sharing-photo-upload-preview"/>
+        <a data-class="facebook"
+           data-url-template="http://www.facebook.com/sharer/sharer.php?u=<%= photoDetailUrl %>"
+           target="_blank"><img src="/static/img/facebook_32.png"></a>
+        <a data-class="twitter"
+           data-url-template="http://twitter.com/share?&text={{ photo_upload_share_text }}&url=<%= photoDetailUrl %>"
+           target="_blank"><img src="/static/img/twitter_32.png"></a>
+        <a data-class="google"
+           data-url-template="https://plus.google.com/share?url=<%= photoDetailUrl %>"
+           target="_blank"><img src="/static/img/googleplus_32.png"></a>
+        <a data-class="pinterest"
+           data-url-template="https://www.pinterest.com/pin/create/button/?&media=<%= photoUrl %>&description={{ photo_upload_share_text }}&url=<%= photoDetailUrl %>"
+           target="_blank"><img src="http://www.seekersofstyle.com/follow_us_pinterest.png"></a>
```

Comment:
You should change this img src to `https:` to avoid a mixed content warning on this page.